sn_node/src/bin/safenode/main.rs

```diff
@@ -296,6 +296,25 @@ fn monitor_node_events(mut node_events_rx: NodeEventsReceiver, ctrl_tx: mpsc::Se
     let _handle = tokio::spawn(async move {
         loop {
             match node_events_rx.recv().await {
+                Ok(NodeEvent::AttemptingNetworkConnection) => {
```

Comment:
Might it be possible to use the existing `PeerAdded` event for this functionality of checking we are connecting to something? Previously that event was actually called `ConnectedToNetwork` I believe.

Reply:
Seems sequential to trigger this marker on the first dial of an initial peer before we get to the point of peers being added to routing table and the ConnectedToNetwork event.